src/libraries/Common/tests/System/Net/Http/Http2LoopbackConnection.cs

```diff
@@ -28,19 +28,21 @@ public class Http2LoopbackConnection : GenericLoopbackConnection
         private readonly TimeSpan _timeout;
         private int _lastStreamId;
         private bool _expectClientDisconnect;
+        private readonly Action<string>? _debugLog;
```

Comment:
Yes, because there was still an ongoing investigation related to a failing test (it was a test problem) -- I will remove them all once I've verified there's no issues remaining